lineman/app/css/print_scss_includes.rb

```diff
@@ -3,5 +3,5 @@
   puts "@import '#{filename}';" unless filename =~ /settings/
 end
 Dir[ File.join('../components', '**', '*.scss') ].each do |filename|
-  puts "@import '#{filename}';" unless filename =~ /settings/
```

Comment:
Hrm. I assume we need this line in order for us to pick up the `email_settings.scss`, but why was it there in the first place? Are there other files which match /settings/ we don't want?